Tests/NSDataMetadataTests.swift

```diff
@@ -67,7 +67,20 @@ class NSDataMetadataTests: XCTestCase {
             XCTAssertNotNil(metadata[String(kCGImagePropertyTIFFDictionary)])
         }
     }
-    
+
+    func testThatGIFSizeDoesNotGrowAfterMetaRemoval() {
```

Comment:
The name of this test is a bit misleading since we just don't do any metadata removal for GIFs.

```suggestion
    func testThatGIFsAreExcludedFromMetadataRemoval() {
```

Reply:
Thanks for the reminder and I will also check other image types also have size grow issue after metadata removal.